src/components/ChatLog.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import './ChatEntry.css';
+import PropTypes from 'prop-types';
+import { DateTime } from 'luxon';
+import ChatEntry from './ChatEntry';
+
+
+
+const ChatLog = ({ entries, updateLike, color }) => {
+    const entryComponents = [];
+    for (const message of entries) {
+        entryComponents.push(
+            <ChatEntry
+                key={message.id}
+                id={message.id}
+                sender={message.sender}
+                body={message.body}
+                timeStamp={message.timeStamp}
+                liked={message.liked}
+                updateLike={updateLike}
+                color={color}
+            />
+        );
+    }
```

Comment:
It's pretty common to use `map` to do this same operation in React code. Feel free to try it out!